tests/integration/migration/morse_claim_supplier_test.go

```diff
@@ -0,0 +1,3 @@
+package migration
+
+// TODO_UPNEXT(@bryanchriswhite, #1034): Add app-integration coverage over claiming Morse accounts as suppliers...
```

Comment:
Not a requirement everywhere, but please use MAINNET where possible.

I'm using it actively to grep for blocking work